operate/client/src/setupProxy.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (app) {
     BASENAME,
     createProxyMiddleware({
       target: `http://localhost:${process.env.IS_E2E ? '8081' : '8080'}`,
-      pathFilter: ['/client-config.js', '/api'],
+      pathFilter: ['/operate/client-config.js', '/api'],
```

Comment:
> I added the base path in [setupProxy.js](https://github.com/camunda/camunda/blob/main/operate/client/src/setupProxy.js) , and tested that is working with `yarn start` 

How did you test it? For me it doesn't work. The middleware matches the request.url in `pathFilter` and redirects to `target`.

So in this case it would match `localhost:3000/operate/client-config.js` and redirect to `localhost:8080/client-config` which is not what we want. 

What we can do is to add a pathRewrite like this: 
```suggestion
      pathFilter: ['/client-config.js', '/api'],
      pathRewrite: {'^/client-config.js': '/operate/client-config.js'}
```

Then Operate works as expected under `localhost:3000/operate`. This is not ideal because `yarn start` opens the browser at `localhost:3000` which is then empty. But I don't have a solution for this right now.

Reply:
It seems I did not test the right thing,  I just checked `http://localhost:3000/client-config.js` in the Dev console, and it seems that it was just rerouted to index

Reply:
@pedesen I solved the `localhost:3000` by adding `/` in `pathFilter`
localhost:3000 -> localhost:8080 -> localhost:8080/operate (the last redirection is done by backend)

Reply:
it works, but then frontend code from the maven build is used (served at port 8080). What we need is the frontend code served by CRA at 3000, so that hot reload still works. So ideally a redirect `localhost:3000/operate` -> `localhost:3000`, but not sure how this is possible using the middleware, because it's a different target port :thinking:

Reply:
I will remove the `/` from the filter

Reply:
done